docs/app.vue

```diff
@@ -281,7 +281,7 @@ export default {
       let emoji = index.findEmoji(':smile:')
       // Note, that position in the emoji sheet is calculated by
       // `emoji` object
-      let style = `background-position: ${emoji.getPosition()}; background-image: url(https://unpkg.com/emoji-datasource-twitter@14.0.0/img/twitter/sheets-256/64.png); width: 24px; height: 24px; display: inline-block; background-size: 5700%`
+      let style = `background-position: ${emoji.getPosition()}; background-image: url(https://unpkg.com/emoji-datasource-apple@15.0.0/img/apple/sheets-256/64.png); width: 24px; height: 24px; display: inline-block; background-size: 5700%`
```

Comment:
❓ Is the `twitter` -> `apple` change functional (something did not work) or just visual (apple looks better?)?

Reply:
I can revert this, mostly this was from a different change (twitter emojis are discontinued and do not provide Emoji 15).